test/uk/gov/hmrc/cdsreimbursementclaim/repositories/FileUploadsRepositorySpec.scala

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//package uk.gov.hmrc.cdsreimbursementclaim.repositories
+//
+//import org.scalamock.scalatest.MockFactory
+//import org.scalatest.matchers.should.Matchers
+//import org.scalatest.wordspec.AnyWordSpec
+//import play.api.inject.DefaultApplicationLifecycle
+//import play.modules.reactivemongo.ReactiveMongoComponentImpl
+//import reactivemongo.bson.BSONObjectID
+//import uk.gov.hmrc.cdsreimbursementclaim.controllers.BaseSpec
+//import uk.gov.hmrc.cdsreimbursementclaim.models.WorkItemPayload
+//import uk.gov.hmrc.http.HeaderCarrier
+//import uk.gov.hmrc.time.DateTimeUtils._
+//import uk.gov.hmrc.workitem.{Failed, PermanentlyFailed, ToDo, WorkItem}
+//
+//import scala.concurrent.duration._
+//import scala.concurrent.{Await, ExecutionContext, Future}
+//
+//class FileUploadsRepositorySpec extends BaseSpec  {
```

Comment:
Remove this if it is not required. Not sure why you are referencing BankAccountReputationService here?